.rubocop.yml

```diff
@@ -18,3 +18,6 @@ Layout/LineLength:
 
 Metrics/BlockLength:
   Enabled: false
+
+Gemspec/RequireMFA:
```

Comment:
Let's remove this, MFA will soon be required for all releases per rubygems

Reply:
Done